components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/serialization_of_types_in_Cairo.adoc

```diff
@@ -0,0 +1,316 @@
+[id="serialization_of_types_in_Cairo"]
+= Serialization of types in Cairo
+
+When you interact with contracts, especially if you are a library or SDK developer that wants to create transactions, you need to understand how Cairo handles types that are larger than 252 bits so you can correctly formulate the calldata in a transaction.
```

Comment:
I would suggest em dashes instead of commas here, but not crucial if you prefer the commas

Reply:
@Mrericsumner Yeah, although the [Google style guide allows them](https://developers.google.com/style/dashes), I've always been able to get around them.